hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/linux/runtime/TestDockerContainerRuntime.java

```diff
@@ -183,20 +182,19 @@ public class TestDockerContainerRuntime {
   private final static String ENV_OCI_CONTAINER_RUN_PRIVILEGED_CONTAINER =
       formatOciEnvKey(RUNTIME_TYPE, RUN_PRIVILEGED_CONTAINER_SUFFIX);
 
-  @Rule
-  public TemporaryFolder tempDir = new TemporaryFolder();
-
-  @Parameterized.Parameters(name = "https={0}")
   public static Collection<Object[]> data() {
     return Arrays.asList(new Object[][] {
         {true}, {false}
     });
   }
 
-  @Parameterized.Parameter
   public boolean https;
 
-  @Before
+  public void initHttps(boolean pHttps) {
```

Comment:
Can this be made `private`?

Reply:
Thank you very much for reviewing the code! I will also pay attention to similar issues in other classes and make improvements.